### ISSUE TITLE
Remove need for libjack and libasound2

### ISSUE THREAD

**Issue by thirtytwobits** [UAVCAN/yakut#23]
There's no good reason why I need to install support for multi-media libraries to run yakut. This makes it difficult to install in thin platforms that don't come with such support. My best guess is that this is coming in through python-rtmidi. I'd suggest either removing that dependency or finding a way to use with without audio support.

**Comment by pavel-kirienko** [comment]
Yes, these are pulled by rtmidi. How about making it available as an extra? Like **`pip install yakut[joystick]`**

**Comment by thirtytwobits** [comment]
I like that idea

**Comment by pavel-kirienko** [comment]
Documenting here that the required packages on Debian-based systems are `libasound2-dev`, `libjack-dev`.

**Comment by thirtytwobits** [comment]
ugh, why should I require multi-media support to use this CLI? We really need a way to make that optional

**Comment by pavel-kirienko** [comment]
@coderkalyan is our only hope.

**Comment by coderkalyan** [comment]
Really sorry for the delay. Will push this to the top of my queue, hopefully in the next day or two.
